specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-04-01/machineLearningServices.json

```diff
@@ -300,6 +300,54 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/prepareNotebook": {
```

Comment:
This change has been added to version 2020-03-01 by this change https://github.com/Azure/azure-rest-api-specs/pull/9058, but 2020-04-01 is missing it. So added the same change to version "2020-04-01"